services/src/main/java/org/keycloak/services/util/ObjectMapperResolver.java

```diff
@@ -34,6 +34,8 @@
 public class ObjectMapperResolver implements ContextResolver<ObjectMapper> {
     protected ObjectMapper mapper = new ObjectMapper();
 
+    public ObjectMapperResolver() {}
```

Comment:
Perhaps it would be better to just change the constructor below to use the system property directly instead of getting "indent" passed from somewhere else?

Reply:
If we just use the `ObjectMapper` instance from `JsonSerialization` instead of creating another instance, do we would have any collateral effect?